client/devdocs/example.js

```diff
@@ -59,7 +59,7 @@ class Example extends Component {
 			...pkgComponents,
 			Component,
 			withState,
-			getSettings,
+			getSettings: __experimentalGetSettings,
```

Comment:
I think you can remove this, since you're not using getSettings in the component anymore.